crates/iroha/tests/integration/set_parameter.rs

```diff
@@ -12,8 +12,10 @@ use iroha_test_network::*;
 
 #[test]
 fn can_change_parameter_value() -> Result<()> {
-    let (_rt, _peer, test_client) = <PeerBuilder>::new().with_port(11_135).start_with_runtime();
-    wait_for_genesis_committed(&vec![test_client.clone()], 0);
+    let (network, _rt) = NetworkBuilder::new()
+        .with_default_pipeline_time()
```

Comment:
No need for this.

Will optimise this test to use some other parameter to check for.